### ISSUE TITLE
Add support for `MD051` rule in MkDocs-flavored admonitions

### ISSUE THREAD

**Issue by sisp** [rvben/rumdl#464]
### Problem or motivation

It seems that (some) rules are not applied to the content of MkDocs-flavored admonitions.

Running `rumdl check` on the sample Markdown below shows no MD051 error although the anchor link doesn't exist:

```console
$ rumdl check test.md
Success: No issues found in 1 file (15ms)
```

But I think the following result is expected:

```console
$ rumdl check test.md
test.md:7:5: [MD051] Link anchor '#two' does not exist in document headings

Issues: Found 1 issues in 1 file (11ms)
```

### Proposed solution

Apply `MD051` (and possibly others as well) to MkDocs-flavored admonitions.

### Feature category

Improve existing rule

### Example markdown

```markdown
# Test MD057

## One

!!! note    

    See [§ two](#two)

## Three

...
```

### Contribution

- [ ] I would be willing to submit a PR to implement this feature

**Comment by rvben** [comment]
This is fixed in v0.1.33. MD051 now detects broken link fragments inside MkDocs admonitions, content tabs, and markdown-enabled HTML blocks when using `--flavor mkdocs` (or with `flavor = "mkdocs"` in config).

```
$ rumdl check --flavor mkdocs test.md
test.md:7:9: [MD051] Link anchor '#two' does not exist in document headings
```

Thanks for reporting this, @sisp!
